[2023-08-11T10:22:20.672+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-08-11T10:22:20.762+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: userCase2.S5 scheduled__2023-08-11T09:00:00+00:00 [queued]>
[2023-08-11T10:22:20.774+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: userCase2.S5 scheduled__2023-08-11T09:00:00+00:00 [queued]>
[2023-08-11T10:22:20.774+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-11T10:22:20.812+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): S5> on 2023-08-11 09:00:00+00:00
[2023-08-11T10:22:20.825+0000] {standard_task_runner.py:57} INFO - Started process 70 to run task
[2023-08-11T10:22:20.832+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'userCase2', 'S5', 'scheduled__2023-08-11T09:00:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/userCase.py', '--cfg-path', '/tmp/tmpjfqliac5']
[2023-08-11T10:22:20.842+0000] {standard_task_runner.py:85} INFO - Job 117: Subtask S5
[2023-08-11T10:22:20.862+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-11T10:22:20.934+0000] {task_command.py:410} INFO - Running <TaskInstance: userCase2.S5 scheduled__2023-08-11T09:00:00+00:00 [running]> on host 484e6be077e8
[2023-08-11T10:22:20.963+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2023-08-11T10:22:21.108+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='userCase2' AIRFLOW_CTX_TASK_ID='S5' AIRFLOW_CTX_EXECUTION_DATE='2023-08-11T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-11T09:00:00+00:00'
[2023-08-11T10:22:21.110+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-11T10:22:21.110+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 10']
[2023-08-11T10:22:21.144+0000] {subprocess.py:86} INFO - Output:
[2023-08-11T10:22:31.142+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-11T10:22:31.175+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=userCase2, task_id=S5, execution_date=20230811T090000, start_date=20230811T102220, end_date=20230811T102231
[2023-08-11T10:22:31.227+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-11T10:22:31.260+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
